spec/rails_helper.rb

```diff
@@ -0,0 +1,70 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
```

Comment:
How do you feel about removing all the comments from this file? They only add unnecessary noise. We want to be able to work on this file and understand it as fast as possible.